keras_retinanet/bin/evaluate.py

```diff
@@ -154,7 +158,7 @@ def main(args=None):
 
     # optionally convert the model
     if args.convert_model:
-        model = models.convert_model(model, anchor_params=anchor_params)
+        model = models.convert_model(model, anchor_params=anchor_params, pyramid_levels = pyramid_levels)
```

Comment:
No spaces.

Reply:
fixed.. I am now using a python linter to hopefully prevent these things =)